credentials/pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <!-- Parent -->
+  <parent>
+    <groupId>xyz.block</groupId>
+    <artifactId>web5-parent</artifactId>
+    <version>0.0.0-main-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <!-- Model Version -->
+  <modelVersion>4.0.0</modelVersion>
+
+  <!-- Artifact Configuration -->
+  <artifactId>web5-credentials</artifactId>
+  <name>Web5 Credentials</name>
+  <description>Credentials libraries for Web5</description>
+
+
+  <!-- Properties -->
+  <properties>
+
+    <!-- Versioning -->
+
+  </properties>
+
+  <!-- Dependencies -->
+  <dependencies>
+
+    <!-- Internal Dependencies -->
+    <dependency>
+      <groupId>xyz.block</groupId>
+      <artifactId>web5-common</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>xyz.block</groupId>
+      <artifactId>web5-crypto</artifactId>
+      <version>${project.version}</version>
```

Comment:
1. where is the project.version declared? (i'm gonna guess at top level pom.xml)
2. unlike JS, we'll bump the version across all packages, even if we update one package, right?

Reply:
1. Yes.
2. Correct, at least this is how we are doing with gradle so far too.